### ISSUE TITLE
login failes when entering 2FA code

### ISSUE THREAD

**Issue by codesmaker** [bitwarden/cli#111]
Hi,

I’m receiving an error when trying to login using the ‘bw’ tool and specifying the 2FA code in the command line as follows:

`bw login myemail mypass --method 2 --code 11111`

But when I enter the code manually as follows:

 ```
bw login myemail mypass --method 2
 ? Two-step login code: 11111
```

Is it a bug or am I missing something?

**Comment by kspearrin** [comment]
Method 2 is Duo, which is not supported by the CLI at this time. Do you need to use method 0 for an authenticator app?

**Comment by codesmaker** [comment]
Thanks for your answer!

OK, it works with method 0. With method 2, it works as well when I enter the 2FA code manually which is strange.

I thought that the method number depends on the order I activated the 2FA methods in the web UI. For example, I have activated two methods: Duo and then an authenticator app. So in my mind, Duo is method 1 and the authenticator app is method 2 but it seems that I was wrong.
